src/core_write.cpp

```diff
@@ -336,7 +336,9 @@ void TxToUniv(const CTransaction& tx, const uint256& hashBlock, bool include_add
     if (calculate_fee) {
         CAmount fee = amt_total_in - amt_total_out;
         if (tx.IsPlatformTransfer()) {
-            fee = CHECK_NONFATAL(GetTxPayload<CAssetUnlockPayload>(tx))->getFee();
+            auto payload = GetTxPayload<CAssetUnlockPayload>(tx);
```

Comment:
nit: missing const

```suggestion
            const auto payload = GetTxPayload<CAssetUnlockPayload>(tx);
```

Reply:
same for other parts

Reply:
meh, it'll be reverted in develop, and compiler can figure that out for this one version :)